src/js/utils/merge-options.js

```diff
@@ -41,7 +43,7 @@ const customizer = function(destination, source) {
  * provided objects
  * @function mergeOptions
  */
-export default function mergeOptions() {
+mergeOptions = function() {
```

Comment:
Guess it doesn't need to change.
I was looking for changes from `function foo` to `foo = function` and got a bit overzealous.